.clang-tidy

```diff
@@ -0,0 +1,108 @@
+
+CheckOptions:
+  - key: readability-identifier-naming.ClassCase
+    value: CamelCase
+
+  - key: readability-identifier-naming.ConstexprVariableCase
+    value: UPPER_CASE
+
+  - key: readability-identifier-naming.EnumCase
+    value: CamelCase
+
+  - key: readability-identifier-naming.EnumConstantCase
+    value: CamelCase
+
+  - key: readability-identifier-naming.FunctionCase
+    value: lower_case
+
+  - key: readability-identifier-naming.GlobalFunctionCase
+    value: lower_case
+
+  - key: readability-identifier-naming.InlineNamespaceCase
+    value: lower_case
+
+  - key: readability-identifier-naming.LocalConstantCase
+    value: lower_case
+
+  - key: readability-identifier-naming.LocalVariableCase
+    value: lower_case
+
+  - key: readability-identifier-naming.MemberCase
+    value: lower_case
+
+  - key: readability-identifier-naming.PrivateMemberPrefix
+    value: '_'
+
+  - key: readability-identifier-naming.ProtectedMemberPrefix
+    value: '_'
+
+  - key: readability-identifier-naming.PublicMemberCase
+    value: lower_case
+
+  - key: readability-identifier-naming.MethodCase
+    value: lower_case
+
+  - key: readability-identifier-naming.PrivateMethodPrefix
+    value: '_'
+
+  - key: readability-identifier-naming.ProtectedMethodPrefix
+    value: '_'
+
+  - key: readability-identifier-naming.NamespaceCase
+    value: lower_case
+
+  - key: readability-identifier-naming.ParameterCase
+    value: lower_case
+
+  - key: readability-identifier-naming.ConstantParameterCase
+    value: lower_case
+
+  - key: readability-identifier-naming.ParameterPackCase
+    value: lower_case
+
+  - key: readability-identifier-naming.StaticConstantCase
+    value: lower_case
+
+  - key: readability-identifier-naming.StaticVariableCase
+    value: lower_case
+
+  - key: readability-identifier-naming.StructCase
+    value: CamelCase
+
+  - key: readability-identifier-naming.TemplateParameterCase
+    value: UPPER_CASE
+
+  - key: readability-identifier-naming.TemplateTemplateParameterCase
+    value: CamelCase
+
+  - key: readability-identifier-naming.TemplateUsingCase
+    value: lower_case
+
+  - key: readability-identifier-naming.TemplateUsingPrefix
+    value: 'u_'
+
+  - key: readability-identifier-naming.TypeTemplateParameterCase
+    value: CamelCase
+
+  - key: readability-identifier-naming.TypedefCase
+    value: CamelCase
+
+  - key: readability-identifier-naming.UnionCase
+    value: CamelCase
+
+  - key: readability-identifier-naming.UnionPrefix
+    value: 'U'
+
+  - key: readability-identifier-naming.UsingCase
+    value: lower_case
+
+  - key: readability-identifier-naming.ValueTemplateParameterCase
+    value: lower_case
+
+  - key: readability-identifier-naming.VariableCase
+    value: lower_case
+
```

Comment:
We should check which items of this list can be removed.

Reply:
I removed the two entries where it's clear that we don't want them.